### ISSUE TITLE
Bitly converted short url start with j.mp instead of amzn.to

### ISSUE THREAD

**Issue by manojsitapara** [impredicative/bitlyshortener#15]
When I tried to convert following url to short URL from bitly website
https://www.amazon.in/Surf-Excel-Matic-Detergent-Powder/dp/B00TS88UZW

I got short url like **https://amzn.to/3ho0Inp** this, but when I convert by using bitlyshortner library, 

I am getting short url like this **https://j.mp/2FbsN2Y**, I know as per documentation this is also bitly shorter, but is it possible to shorter link as similar we get on bitly website? 

I am able to achieve this by using their API in my python code, but I really like this library, so curious to know is it possible to do similar in this library?

**Comment by impredicative** [comment]
Acknowledged. For the moment, for such Amazon links, you can manually use the following which should work:

    short_url.replace("https://j.mp/", "https://amzn.to/", 1)

**Comment by manojsitapara** [comment]
Thank you for your reply !!! 

The above solution may not work for me because I am doing URL shorter for other website also, which will end up converting with "https://j.mp"

Shorter URL starts with "https://amzn.to" only when full URL is amazon only. for all rest URL, it should start with "https://j.mp/"

**Comment by impredicative** [comment]
Until this is implemented, you can use the following:
```python
if long_url.startswith("https://amazon."):
    short_url = short_url.replace("https://j.mp/", "https://amzn.to/", 1)
```

**Comment by manojsitapara** [comment]
Okay. Thank you for this wonderful library.

**Comment by impredicative** [comment]
This is now implemented. Refer to the optional `vanitize=True` feature documented in the readme.

**Comment by manojsitapara** [comment]
Thanks !! I will check it out.

**Comment by impredicative** [comment]
FYI: A bug was fixed in v0.4.2.

**Comment by impredicative** [comment]
v0.5.0 always in effect uses the vanitize mode. That's because forcing the j.mp or bit.ly URLs for vanity domains is no longer supported by Bitly. Refer to the examples in the readme.
